Content.Shared/Labels/Components/HandLabelerComponent.cs

```diff
@@ -1,6 +1,7 @@
+
 using Content.Shared.Whitelist;
 
-namespace Content.Server.Labels.Components
+namespace Content.Shared.Labels.Components
 {
```

Comment:
if you are gonna move it to shared:
1. actually predict it move the system there too and make it play nicely
2. file scope namespace
3. replace each fields attribute with just `[DataField]` the rest has been automatically done for some time
4. remove `{ get; set; }` and add access attribute for the system
5. probably add `NetworkedComponent`

Reply:
Thanks for reviewing this commit, and giving concrete points to work on.
I'll get to it tomorrow, since figuring all of that out properly isn't that doable after a long day of trying to learn stuff.